### ISSUE TITLE
Struggling to change the text of drop-down items

### ISSUE THREAD

**Issue by anup-singh** [OvidijusParsiunas/deep-chat#170]
How to change the text of menu items "Photo" & "Image"

![image](https://github.com/OvidijusParsiunas/deep-chat/assets/6386991/6cc77cba-5b97-4924-9450-45885d717a8d)

**Comment by OvidijusParsiunas** [comment]
Hi @anup-singh.

Deep Chat currently only holds an option to overwrite full menu items via the [`dropupStyles`](https://deepchat.dev/docs/styles/#dropupStyles) property. This means that you have to provide both - the svg and the text. If you use text by itself - the item will only contain text. The following is an example for [images](https://deepchat.dev/docs/files#images):

```
{
  button: {
    styles: {
      text: {
        content: 'Images',
      },
    },
  },
}
```
You would also need to use the other properties in [`dropupStyles`](https://deepchat.dev/docs/styles/#dropupStyles) to adjust the overall styling.

I will try to investigate if I can implement a better way to only overwrite text or only overwrite SVGs. The next couple of days will be quite busy for me but please bear with me and I will update you on the latest progress of my research.

Thankyou!

**Comment by OvidijusParsiunas** [comment]
Hi, apologies for the wait. You can now set the text of dropdown items without affecting the icons in `deep-chat-dev` and `deep-chat-react-dev` packages version `9.0.171`.

They `dev` packages behave just like the main packages except their names are different. They also contain a few property name changes, such as the [request](https://deepchat.dev/docs/connect#request) property is changed to connect and [stream](https://deepchat.dev/docs/connect#stream) is moved to the connect object. All of the current/old properties will still keep working as usual except you will see warnings in the console for their deprecation and get a few TypeScript warnings which you can ignore.

Let me know if this helps!

**Comment by OvidijusParsiunas** [comment]
Hi, the functionality discussed in this issue has now been released in Deep Chat version `2.0.0`.
